jobs/keepalived/spec

```diff
@@ -10,10 +10,12 @@ packages:
 
 templates:
   bin/keepalived_ctl.erb: 		bin/keepalived_ctl
+  bin/ha_health_check_ctl.erb:          bin/ha_health_check_ctl
   bin/action.sh.erb: 			bin/action.sh
   bin/notify.sh.erb: 			bin/notify.sh
   bin/get_keepalived_state.sh.erb: 	bin/get_keepalived_state.sh
   bin/get_sf_broker_status.sh.erb: 	bin/get_sf_broker_status.sh
+  bin/ha_health_check.js:               bin/ha_health_check.js
```

Comment:
indentation!

Reply:
if you view the file, the indentation is coming proper.. dunno why its showing in git differently.. :(